spec/unit/knife/bootstrap_options_spec.rb

```diff
@@ -84,6 +84,9 @@ def compare_property(sym)
       :winrm_password => :ssh_password,
       :winrm_port => :ssh_port,
       :winrm_ssl_verify_mode => :host_key_verify,
+      :bootstrap_vault_file => :bootstrap_vault_file,
+      :bootstrap_vault_item => :bootstrap_vault_item,
+      :bootstrap_vault_json => :bootstrap_vault_json,
```

Comment:
You shouldn't need these mappings, these are only for when the options are different.